docs/learn/documentation/versioned/yarn/yarn-resource-localization.md

```diff
@@ -18,58 +18,49 @@ title: YARN Resource Localization
+For Samza jobs running on YARN, resource localization leverages the YARN node manager's localization service. Here is a [deep dive](https://hortonworks.com/blog/resource-localization-in-yarn-deep-dive/) on how localization works in YARN. 
 
-If there is an implementation of [FileSystem](https://hadoop.apache.org/docs/stable/api/index.html?org/apache/hadoop/fs/FileSystem.html) on YARN supporting a scheme, then that scheme can be used for resource localization. 
+Depending on where and how the resource comes from, fetching the resource is associated with a scheme in the path (such as `http`, `https`, `hdfs`, `ftp`, `file`, etc). The scheme maps to a corresponding `FileSystem` implementation for handling the localization. 
 
-There are some predefined file systems in Hadoop or Samza, which are provided if you run Samza jobs on YARN:
+There are some predefined `FileSystem` implementations in Hadoop or Samza, which are provided if you run Samza jobs on YARN:
 
-* `org.apache.samza.util.hadoop.HttpFileSystem`: used for fetching resources based on http, or https without client side authentication requirement.
+* `org.apache.samza.util.hadoop.HttpFileSystem`: used for fetching resources based on http, or https without client side authentication.
 * `org.apache.hadoop.hdfs.DistributedFileSystem`: used for fetching resource from DFS system on Hadoop.
 * `org.apache.hadoop.fs.LocalFileSystem`: used for copying resources from local file system to the job directory.
 * `org.apache.hadoop.fs.ftp.FTPFileSystem`: used for fetching resources based on ftp.
-* ...
 
-If you would like to have your own file system, you need to implement a class which extends from `org.apache.hadoop.fs.FileSystem`. 
+If you would like to have your own file system, you should implement a class which extends from `org.apache.hadoop.fs.FileSystem`. 
 
-### Job Configuration
-With the configuration properly defined, the resources a job requiring from external or internal locations may be prepared automatically before it runs.
-
-For each resource with the name `<resourceName>` in the Samza job, the following set of job configurations are used when running on a YARN cluster. The first one which definiing resource path is required, but the others are optional and they have default values.
+### Resource Configuration
+You can specify a resource to be localized by the following configuration.
 
+#### Required Configuration
 1. `yarn.resources.<resourceName>.path`
-    * Required
-    * The path for fetching the resource for localization, e.g. http://hostname.com/packages/mySamzaJob
+    * The path for fetching the resource for localization, e.g. http://hostname.com/packages/myResource
+
+#### Optional Configuration
 2. `yarn.resources.<resourceName>.local.name`
```

Comment:
Yeah, I mean let's describe it there. Otherwise the documentation reads like: resource.local.name: local name of the resource.